app/src/main/res/layout/activity_education.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".EducationActivity">
+
+    <TextView
+        android:id="@+id/eduTitle"
+        android:layout_width="318dp"
+        android:layout_height="80dp"
+        android:layout_marginTop="38dp"
+        android:layout_marginBottom="37dp"
+        android:text="Education and Skills"
```

Comment:
We can either use @ string resources to use strings or hardcode strings like is being done here. I generally think it's a pain in the ass to keep creating string resources (although apparently it's a bad practice). With that being said, I think it's fine to do this for now as it's main uses appear to be localization for consistency and also it helps with translations if we ever need to release the app in a different language which is doubtful. But thought I'd make a comment about this anyway so everyone's on the same page.